netlify.toml

```diff
@@ -11,7 +11,13 @@ NODE_VERSION = "22.15.0"
 # this path should not be changed as various external sites depend on it for OPA
 # badges.
 path = "/badge-endpoint/*"
-function = "badge-endpoint"
+function = "badge"
+
+[[edge_functions]]
+# this path should not be changed as various external sites depend on it for OPA
+# badges.
+path = "/badge/*"
+function = "badge"
```

Comment:
This makes the function match this path too, sadly there is no 'paths' option as far as I can tell.